hillel-expense-tracker-repo/src/main/java/ua/ithillel/expensetracker/Application.java

```diff
@@ -0,0 +1,7 @@
+package ua.ithillel.expensetracker;
+
+public class Application {
```

Comment:
use package-info file instead of Application class. Example https://www.baeldung.com/java-package-info